### PR TITLE
[#IP-306] Handle Cosmos unix timestamp format

### PR DESCRIPTION
**Pull request by AleDore**
#### List of Changes

* update default value of `optOutEmailSwitchDate`
* add specific tests

#### Motivation and Context

This solve a problem while comparing a Date made from a Unix Timestamp and another one made from Timestamp expressed in millisecond.

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: StoreMessageContentActivity/__tests__/handler.test.ts [hunk 1]
```diff
@@ -70,13 +70,15 @@ const lServicePreferencesModel = ({
...
 
-const aPastOptOutEmailSwitchDate = UTCISODateFromString.decode(
-  "2000-07-08T23:59:59Z"
-).getOrElseL(() => fail("wrong date value"));
+const aPastOptOutEmailSwitchDate = new Date(nowTimestamp - 10);
+
+console.log(nowTimestamp);
+console.log(aFutureOptOutEmailSwitchDate);
+console.log(aPastOptOutEmailSwitchDate);
+console.log(isBefore(nowTimestamp, aFutureOptOutEmailSwitchDate));
```
Comment (balanza) [inline StoreMessageContentActivity/__tests__/handler.test.ts:81]: do we need these logs?

**Review by balanza — APPROVED** [review]

File: StoreMessageContentActivity/__tests__/handler.test.ts [hunk 2]
```diff
@@ -70,13 +70,15 @@ const lServicePreferencesModel = ({
   find: findServicePreferenceMock
 } as unknown) as ServicesPreferencesModel;
 
-const anOptOutEmailSwitchDate = UTCISODateFromString.decode(
-  "2021-07-08T23:59:59Z"
-).getOrElseL(() => fail("wrong date value"));
+const nowTimestamp = Math.floor(new Date().getTime() / 1000);
```
Comment (BurnedMarshal) [inline StoreMessageContentActivity/__tests__/handler.test.ts:73]:
```suggestion
const lastUpdateTimestamp = Math.floor(new Date().getTime() / 1000);
```

**Review by BurnedMarshal — CHANGES_REQUESTED** [review]

File: StoreMessageContentActivity/__tests__/handler.test.ts (hunk 1 above)
Comment (AleDore) [reply StoreMessageContentActivity/__tests__/handler.test.ts:81]: Nope I'll remove it

**Review by BurnedMarshal — APPROVED** [review]
LGTM

**Pull request merged**